### ISSUE TITLE
com.baidu.cloud:spring-cloud-baidu-thirdparty-commons:jar:2022.2.0 与 Netty Epoll 冲突

### ISSUE THREAD

**Issue by TeslaCN** [baidu/starlight#360]
**Describe the bug**

https://github.com/apache/shenyu/issues/5033

`java.lang.IllegalStateException: Multiple resources found for 'META-INF/native/libnetty_transport_native_epoll_x86_64.so' with different content: [jar:file:/home/wuweijie/.m2/repository/io/netty/netty-transport-native-epoll/4.1.94.Final/netty-transport-native-epoll-4.1.94.Final-linux-x86_64.jar!/META-INF/native/libnetty_transport_native_epoll_x86_64.so, jar:file:/home/wuweijie/.m2/repository/com/baidu/cloud/spring-cloud-baidu-thirdparty-commons/2022.2.0/spring-cloud-baidu-thirdparty-commons-2022.2.0.jar!/META-INF/native/libnetty_transport_native_epoll_x86_64.so]`

![image](https://github.com/baidu/starlight/assets/20503072/85087dd9-9e24-41d2-8dee-313b89aa0ebd)

**To Reproduce**
2. See error

**Expected behavior**

**Additional context**

**Comment by Reason94** [comment]
Thanks for feedback, we will solve it.

The solution is：
1. move META-INF/native/libnetty_transport_native_epoll_x86_64.so  in spring-cloud-baidu-thirdparty-commons to new path
2. starlight will load libnetty_transport_native_epoll_x86_64.so from new path

**Comment by Reason94** [comment]
> Thanks for feedback, we will solve it.
> 
> The solution is：
> 
> 1. move META-INF/native/libnetty_transport_native_epoll_x86_64.so  in spring-cloud-baidu-thirdparty-commons to new path
> 2. starlight will load libnetty_transport_native_epoll_x86_64.so from new path

https://github.com/baidu/starlight/pull/366

**Comment by Reason94** [comment]
Already fixed in version 2022.2.2-SNAPSHOT，please verify and use
```xml
<dependency>
  <groupId>com.baidu.cloud</groupId>
  <artifactId>spring-cloud-starter-baidu-starlight</artifactId>
  <version>2022.2.2-SNAPSHOT</version>
</dependency>
```
